plasmapy/simulation/particle_tracker.py

```diff
@@ -169,6 +170,89 @@ def total(self) -> float:
         return float(self.tracker.nparticles)
 
 
+class ParticlesPushedThroughGridsTerminationCondition(AbstractTerminationCondition):
+    """Termination condition corresponding to stopping the simulation when a provided
+    proportion of particles have entered and exited the grids.
```

Comment:
```suggestion
    fraction of particles have entered and exited the grids.
```

to match the language in the keywords 'fraction'